lib/between_meals/knife.rb

```diff
@@ -242,8 +244,10 @@ def write_user_config
       BLOCK
 
       begin
-        unless File.directory?(File.dirname(@pem))
+        begin
           Dir.mkdir(File.dirname(@pem), 0o755)
+        rescue SystemCallError
+          raise unless File.directory?(File.dirname(@pem))
         end
       rescue Errno::EEXIST => e
```

Comment:
theres' already a rescue here... specifically for the case where it already exists....

Reply:
Ah yes, that is mostly useless in practice. I'll clean it up.